README.md

```diff
@@ -21,21 +23,113 @@ cd path/to/application
 bundle
 ```
 
-Then, configure your secrets.
+Then, add your client ID and secret to Workarea. See "Account Setup" below for more details.
+
+For production environments, we recommend you run the webhooks rake task to register critical callbacks from PayPal to help keep the state of your orders on Workarea inline with the state of the PayPal transaction. See the `default_webhook_events` description under "Configuration Options" for more information.
+
+```bash
+bin/rails workarea:paypal:create_webhooks
+```
+
+Account Setup
+--------------------------------------------------------------------------------
+
+To begin using PayPal on your site, you will need to have a PayPal merchant account. For development, you can create an account on [developer.paypal.com](htttps://developer.paypal.com). From there you can create a sandbox REST API app, and use those credentials for any non-live environments.
+
+Once you have a REST API application in your paypal sandbox, you must add the client ID and secret to your Workarea application. This can be done through admin configuration, environment variables, rails credentials, or hard-coded Workarea configuration.
```

Comment:
```suggestion
Once you have a REST API application in your PayPal sandbox, you must add the client ID and secret to your Workarea application. This can be done through admin configuration, environment variables, Rails credentials, or hard-coded Workarea configuration.
```